src/llmperf/ray_clients/openai_chat_completions_client.py

```diff
@@ -59,6 +59,12 @@ def llm_request(self, request_config: RequestConfig) -> Dict[str, Any]:
         if not address.endswith("/"):
             address = address + "/"
         address += "chat/completions"
+        if "azure"in address:
```

Comment:
I suggest to replace `"azure"` by `"openai.azure.com"` because we can use other model (for example Mistral Large) in `"ml.azure.com"` and we don't want enter in this bloc of code.